media-video/pipewire/pipewire-9999.ebuild

```diff
@@ -172,8 +178,8 @@ multilib_src_configure() {
 		$(meson_feature dbus)
 		$(meson_native_use_feature zeroconf avahi)
 		$(meson_native_use_feature doc docs)
-		$(meson_native_enabled examples) # TODO: Figure out if this is still important now that media-session gone
-		$(meson_native_enabled man)
+		$(meson_native_use_feature examples) # TODO: Figure out if this is still important now that media-session gone
```

Comment:
Please justify the examples flag here. How big is the delta before/after? Pipewire already has a tonne of USE flags.